global/data/example-data.json

```diff
@@ -32,7 +32,20 @@
             "users_email_subject": "OpenSlides access data",
             "users_email_body": "Dear {name},\n\nthis is your personal OpenSlides login:\n\n{url}\nUsername: {username}\nPassword: {password}\n\n\nThis email was generated automatically.",
             "url": "https://example.com",
-            "user_ids": [1, 2, 3]
+            "user_ids": [1, 2, 3],
+            "sso_enabled": false,
+            "login_button_text": "SAML Login",
+            "save_attr_config": {
+                "username": "saml_id",
+                "title": "title",
+                "firstName": "first_name",
+                "lastName": "last_name",
+                "email": "email",
+                "gender": "gender",
+                "pronomen": "pronoun",
+                "is_active": "is_active",
+                "is_person": "is_physical_person"
+              }
```

Comment:
This is the only part in the software to give an example and to document what and how to configure the mapping of attributes. That's the reason why I put it here. It could be hidden in the client, but if it will be enabled these base settings should be there.

Reply:
It does not need to be documented in the software. We should add a hint in the client when enabling SSO as well as document this behaviour in some README, but since we will be the ones to enter these details for almost all instances, it does not need to be present in the datastore of all instances when 99% of them don't use the feature.